pkg/controller/controller_binding_test.go

```diff
@@ -296,19 +298,19 @@ func TestReconcileServiceInstanceCredentialWithSecretConflict(t *testing.T) {
 	})
 
 	actions := fakeCatalogClient.Actions()
-	assertNumberOfActions(t, actions, 2)
+	assertNumberOfActions(t, actions, 1)
```

Comment:
There should be 2 actions here.

1. Status update to set current operation.
2. Status update to indicate that the credential injection failed.